app/controllers/users_controller.rb

```diff
@@ -6,10 +6,29 @@ def index
 		@users = User.all.paginate(page: params[:page], per_page: 3)
 	end
 
+	def show
+		@user = User.find(params[:id])
+	end
+
 	def new
 		@user = User.new
 	end
 
+	def edit
+		@user = User.find(params[:id])
+	end
+
+	def update
+		@user = User.find(params[:id])
+		if
+			@user.update_attributes(user_params)
+			redirect_to user_url
+			flash[:success] = "SUCCESS EDIT"
```

Comment:
Place the flash message before the redirect statement.

flash[:success] = "SUCCESS EDIT"
redirect_to user_url